libapicore/ApiServer.cpp

```diff
@@ -624,6 +624,30 @@ void ApiConnection::processRequest(Json::Value& jRequest, Json::Value& jResponse
         jResponse["result"] = true;
     }
 
+    else if (_method == "miner_setverbosity")
+    {
+        if (!checkApiWriteAccess(m_readonly, jResponse))
+            return;
+
+        Json::Value jRequestParams;
+        if (!getRequestValue("params", jRequestParams, jRequest, false, jResponse))
+            return;
+
+        unsigned verbosity;
+        if (!getRequestValue("verbosity", verbosity, jRequestParams, false, jResponse))
+            return;
+
+        if (verbosity > 9)
```

Comment:
Would suggest to check bounds 0 <-> 9 to avoid possible issues in setting negative values.

if (verbosity < 0 || verbosity > 9)

Reply:
As `verbosity` is unsigned I thing it can not be "< 0" ;-)

Reply:
Hm... we should settle if we interpret the json request as int or unsigned!!!
If you look at `miner_setactiveconnection`, `miner_removeconnection` or `miner_pausegpu` we interpret the index as an unsigned.
Not sure if we should make some specials for `miner_setverbosity` ?

Reply:
Agree ... in fact I believe on those methods you mentioned I have made an error not foreseeing this condition.